hawkbit-ui/src/main/resources/messages_de.properties

```diff
@@ -159,6 +159,7 @@ label.tag.name = Tag name
 label.configuration.auth.header = Allow targets to authenticate via a certificate authenticated by an reverse proxy
 label.configuration.auth.gatewaytoken = Allow a gateway to authenticate and manage multiple targets through a gateway security token
 label.configuration.auth.targettoken = Allow targets to authenticate directly with their target security token
+label.configuration.anonymous.download = Allow targets to download anonymous
```

Comment:
The label sentence must be `Allow targets to download anonymously` but I guess I would change the sentence more like `Allow targets to download artifacts without security credentials` @kaizimmerm  would you think?